egs/sre08/v1/sid/nnet3/xvector/get_egs.sh

```diff
@@ -0,0 +1,241 @@
+#!/bin/bash
+
+# Copyright      2017 Johns Hopkins University (Author: Daniel Povey)
+#                2017 Johns Hopkins University (Author: Daniel Garcia-Romero)
+#                2017 David Snyder
+# Apache 2.0
+#
+# This script dumps training examples (egs) for multiclass xvector training.
+# These egs consist of a data chunk and a label. Each archive of egs has,
```

Comment:
explain that the label corresponds to the speaker, and whether it's one-based or zero-based.